recipes/_source.rb

```diff
@@ -36,7 +36,7 @@
 pdns_source_dir = pdns_dir(pdns_filename)
 
 remote_file pdns_filepath do
-  source lazy { node['pdns'][flavor]['source']['url'] }
+  source node['pdns'][flavor]['source']['url']
```

Comment:
I'm using Chef 11.10, which is the most recent Chef 11.x that I can get on AWS OpsWorks :-/